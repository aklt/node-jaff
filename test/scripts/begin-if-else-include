#begin coffee

get = 'if-else'

#end
Getting {get}
#if false
#include {get}-false
#else
#include {get}-true
#end
And that was it
