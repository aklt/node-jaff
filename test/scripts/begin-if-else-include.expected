Getting if-else
before
branch 1
after
And that was it
